clang/test/CodeGenSYCL/free_function_default_template_arguments.cpp

```diff
@@ -267,17 +267,17 @@ namespace Testing::Tests {
 // CHECK-NEXT:   return (void (*)(struct ns::Arg<class ns::ns1::hasDefaultArg<struct ns::notatuple>, int, 12, struct ns::notatuple>))simple1;
 // CHECK-NEXT: }
 
-// CHECK: template <typename T> void templated(ns::Arg<T, float, 3, ns::notatuple> , T end);
+// CHECK: template <typename T> void templated(ns::Arg<T, float, 3, ns::notatuple, <>>, T);
```

Comment:
Two changes here:
- names of arguments in forward-declarations were dropped intentionally, because that's unnecessary information
- `<>` is the way how empty type packs are printed. I'm not 100% sure that this is a legal C++ code, but I checked that generated integration header compiles with clang